### DISCUSSION TITLE
Failed to Initialize Reasonable Values during CAFE Execution

### DISCUSSION THREAD

**Discussion by meeranhussain** [hahnlab/CAFE5#194] [Q&A] [answered]
Hi,

I am encountering an issue while running CAFE, where the analysis fails to initialize any reasonable values. The output shows the families with the largest size differentials, as follows:

"Families with largest size differentials:
OG0000008: 88
OG0000002: 82
OG0000001: 79
OG0000009: 75
OG0000012: 69
OG0000073: 68
OG0000000: 68
OG0000196: 67
OG0000007: 65
OG0000191: 63
OG0000003: 57
OG0000038: 53
OG0000014: 53
OG0000200: 51
OG0000034: 51
OG0000006: 50
OG0000308: 50
OG0000013: 49
OG0000037: 48
OG0000047: 48

You may want to try removing the top few families with the largest difference
between the max and min counts and then re-run the analysis.

Failed to initialize any reasonable values"

The message suggests removing the top few families with the largest difference between the max and min counts and re-running the analysis. I tried removing those families, but it did not resolve the issue. I also explored similar issues on GitHub but couldn't find any solutions that worked in my case.

I have attached the **ultrametric tree and cafe_input.tsv** files below for your reference. Could you kindly provide any suggestions or guidance to help resolve this issue?

ultrametric_tree: "(((Orussus_abietinus:0.242585,Athalia_rosae:0.242585):0.0270296,(Solenopsis_invicta:0.218233,Apis_mellifera:0.218233):0.0513808):0.0164564,(Trichogramma_pretiosum:0.272074,((Microctonus_hyperodae:0.0407805,(Microctonus_aethiopoides_USA:0.00652269,(Microctonus_aethiopoides_WAL:0.0020169,(Microctonus_aethiopoides_SCO:0.00195439,(Microctonus_aethiopoides_ROM:0.00194631,((Microctonus_aethiopoides_NOR:0.00189239,(Microctonus_aethiopoides_MOR:0.00189112,(Microctonus_aethiopoides_IRE:0.00186727,(Microctonus_aethiopoides_ENG:0.00183455,Microctonus_aethiopoides_FRN:0.00183455):3.27195e-05):2.38499e-05):1.27e-06):4.93827e-05,Microctonus_aethiopoides_NZI:0.00194177):4.54358e-06):8.07317e-06):6.25109e-05):0.0045058):0.0342578):0.153079,(Cotesia_glomerata:0.0856837,Microplitis_demolitor:0.0856837):0.108176):0.0782141):0.0139969);" 

Thank you in advance for your support.

[cafe.input.tsv.gz](https://github.com/user-attachments/files/17468441/cafe.input.tsv.gz)
[SpeciesTree_rooted.txt.ultrametric.tre.gz](https://github.com/user-attachments/files/17468445/SpeciesTree_rooted.txt.ultrametric.tre.gz)

Cheers,
Meeran

**Comment by benfulton** [comment, marked as answer]
Your branch lengths may be too short. See #152 for some discussion of that issue.

**Comment by meeranhussain** [comment]
It worked! Thank you so much :)
